test/apps/iodemo/io_demo.cc

```diff
@@ -584,68 +611,99 @@ class DemoClient : public P2pDemoCommon {
     }
 
     bool run() {
-        std::vector<UcxConnection*> conn;
-        conn.resize(opts().servers.size());
-        for (size_t i = 0; i < conn.size(); i++) {
-            conn[i] = connect(opts().servers[i]);
-            if (!conn[i]) {
-                LOG << "Connect to server ["
-                    << opts().servers[i]
-                    << "] Failed!";
-                for (size_t j = 0; j < i; j++) {
-                    delete conn[j];
+        if (_conn.size() == 0) {
+            for (size_t i = 0; i < opts().servers.size(); i++) {
+                conn_info_t conn_info = {0, 0, NULL};
+                _conn.push_back(conn_info);
```

Comment:
minor: can be simplified
```
_conn.resize(_conn.size() + 1);
```
the result will be the same since C++ will memset(0) in case of lack default constructor

Reply:
@evgeny-leksikov Changed the the code to ``_conn.resize(opts().servers.size());``. Is it right?

Reply:
should work